e2e/tests/i18n/editview.spec.ts

```diff
@@ -0,0 +1,238 @@
+import { test, expect } from '@playwright/test';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+import { login } from '../../utils/login';
+
+test.describe('Edit view', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath('./e2e/data/with-admin.tar');
+    await page.goto('/admin');
+    await login({ page });
+  });
+
+  test('As a user I want to create a brand new document in the non-default locale', async ({
+    page,
+  }) => {
+    const LIST_URL = /\/admin\/content-manager\/collection-types\/api::product.product(\?.*)?/;
+    const EDIT_URL =
+      /\/admin\/content-manager\/collection-types\/api::product.product\/[^/]+(\?.*)?/;
+    const CREATE_URL =
+      /\/admin\/content-manager\/collection-types\/api::product.product\/create(\?.*)?/;
+
+    /**
+     * Navigate to our products list-view
+     */
+    await page.getByRole('link', { name: 'Content Manager' }).click();
+    await page.getByRole('link', { name: 'Products' }).click();
+    await page.waitForURL(LIST_URL);
+    await expect(page.getByRole('heading', { name: 'Products' })).toBeVisible();
+
+    /**
+     * Assert we're on the english locale and our document exists
+     */
+    await expect(page.getByRole('combobox', { name: 'Select a locale' })).toHaveText(
+      'English (en)'
+    );
+    await expect(
+      page.getByRole('row', { name: 'Nike Mens 23/24 Away Stadium Jersey' })
+    ).toBeVisible();
+
+    /**
+     * Swap to es locale to create a new document
+     */
+    await page.getByRole('combobox', { name: 'Select a locale' }).click();
+    await page.getByRole('option', { name: 'Spanish (es)' }).click();
+    await expect(page.getByRole('combobox', { name: 'Select a locale' })).toHaveText(
+      'Spanish (es)'
+    );
+    await expect(page.getByRole('row', { name: 'No content found' })).toBeVisible();
+
+    /**
+     * So now we're going to create a document.
+     */
+    await page.getByRole('link', { name: 'Create new entry' }).first().click();
+    await page.waitForURL(CREATE_URL);
+    await expect(page.getByRole('heading', { name: 'Create an entry' })).toBeVisible();
+    await expect(page.getByRole('combobox', { name: 'Locales' })).toHaveText('Spanish (es)');
+    expect(new URL(page.url()).searchParams.get('plugins[i18n][locale]')).toEqual('es');
+    await page
+      .getByRole('textbox', { name: 'name' })
+      .fill('Camiseta de fuera 23/24 de Nike para hombres');
+
+    /**
+     * Verify the UID works as expected
+     */
+    await expect
+      .poll(async () => {
+        const requestePromise = page.waitForRequest('**/content-manager/uid/generate?locale=es');
```

Comment:
```suggestion
        const requestPromise = page.waitForRequest('**/content-manager/uid/generate?locale=es');
```

Or maybe requeste is spanish? :D